src/components/services/AccountCallbackManager.tsx

```diff
@@ -63,8 +64,8 @@ export function AccountCallbackManager(props: Props) {
       }),
 
       watchSecurityAlerts(account, hasAlerts => {
-        if (hasAlerts && Actions.currentScene !== 'securityAlerts') {
-          Actions.push('securityAlerts', {})
+        if (hasAlerts && navigation.state.currentScene !== 'securityAlerts') {
```

Comment:
`navigation.state.currentScene` is not going to be a thing. Sorry.. This was one of the things that we decided we're going to keep as `Actions.` API left-over until the flip to the new lib.